src/Autofac/Autofac.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <Description>Autofac is an IoC container for Microsoft .NET. It manages the dependencies between classes so that applications stay easy to change as they grow in size and complexity.</Description>
     <VersionPrefix>5.0.0</VersionPrefix>
-    <TargetFrameworks>netstandard2.0;netstandard1.1;net45</TargetFrameworks>
+    <TargetFrameworks>netcoreapp3.0;netstandard2.0;netstandard1.1;net45</TargetFrameworks>
```

Comment:
There's an ns2,0 package that carries these interfaces https://www.nuget.org/packages/Microsoft.Bcl.AsyncInterfaces/1.1.0-preview1.19504.10

Reply:
Thanks @davidfowl, I've changed to include the linked package for netstandard2.0, and add target for netstandard2.1, which won't require that package.